packages/beacon-node/src/chain/archiver/strategies/frequencyStateArchiveStrategy.ts

```diff
@@ -1,34 +1,23 @@
 import {Logger} from "@lodestar/utils";
 import {SLOTS_PER_EPOCH} from "@lodestar/params";
-import {Slot, Epoch} from "@lodestar/types";
+import {Slot, Epoch, RootHex} from "@lodestar/types";
 import {computeEpochAtSlot, computeStartSlotAtEpoch} from "@lodestar/state-transition";
 import {CheckpointWithHex} from "@lodestar/fork-choice";
-import {IBeaconDb} from "../../db/index.js";
-import {IStateRegenerator} from "../regen/interface.js";
-import {getStateSlotFromBytes} from "../../util/multifork.js";
-import {serializeState} from "../serializeState.js";
-import {AllocSource, BufferPool} from "../../util/bufferPool.js";
-import {Metrics} from "../../metrics/metrics.js";
-
-/**
- * Minimum number of epochs between single temp archived states
- * These states will be pruned once a new state is persisted
- */
-const PERSIST_TEMP_STATE_EVERY_EPOCHS = 32;
-
-export interface StatesArchiverOpts {
-  /**
-   * Minimum number of epochs between archived states
-   */
-  archiveStateEpochFrequency: number;
-}
+import {IBeaconDb} from "../../../db/index.js";
+import {IStateRegenerator} from "../../regen/interface.js";
+import {getStateSlotFromBytes} from "../../../util/multifork.js";
+import {serializeState} from "../../serializeState.js";
+import {AllocSource, BufferPool} from "../../../util/bufferPool.js";
+import {Metrics} from "../../../metrics/metrics.js";
+import {StateArchiveStrategy, StatesArchiverOpts} from "../interface.js";
+import {PERSIST_TEMP_STATE_EVERY_EPOCHS} from "../constants.js";
```

Comment:
I prefer to have constants specific to this file's strategy to stay on this file